rust/feature-flags/src/api/endpoint.rs

```diff
@@ -53,6 +53,12 @@ pub async fn flags(
     Ok(Json(process_request(context).await?))
 }
 
+pub async fn options() -> Result<Json<FlagsOptionsResponse>, FlagError> {
+    Ok(Json(FlagsOptionsResponse {
+        status: FlagsResponseCode::Ok,
+    }))
+}
```

Comment:
handle options requests like we do in `/decide` (see: https://github.com/PostHog/posthog/blob/feat/flags-to-prod/posthog/api/decide.py#L167-L169)